examples/resources/awscc_elasticloadbalancingv2_target_group/elasticloadbalancingv2_target_group_instance.tf

```
@@ -0,0 +1,24 @@
+resource "awscc_elasticloadbalancingv2_target_group" "instance-example" {
+  name     = "instance-example"
+  port     = 80
+  protocol = "HTTP"
+  vpc_id   = awscc_ec2_vpc.main.id
+  tags = [{
+    key   = "Modified By"
+    value = "AWSCC"
+  }]
+}
+
+resource "awscc_ec2_vpc" "main" {
+  cidr_block = "10.0.0.0/16"
+  tags = [
+    {
+      key   = "Name"
+      value = "main"
+    },
+    {
+      key   = "Modified By"
+      value = "AWSCC"
+    }
+  ]
```

Keep the tags consistent with the rest of resources

```suggestion
  tags = [
    {
      key   = "Modified By"
      value = "AWSCC"
    }
  ]
```

---

i was using the `Name` tag to give a name to the VPC resource, which can only be given through tags, but it's alright for this example vpc to be nameless.